packages/arb-token-bridge-ui/tests/e2e/specs/depositERC20.cy.ts

```diff
@@ -74,13 +91,13 @@ describe('Deposit ERC20 Token', () => {
             .should('be.visible')
             .click({ scrollBehavior: false })
 
-          // Select the LINK token
-          cy.findByText('ChainLink Token').click({ scrollBehavior: false })
+          // Select the WETH token
+          cy.findAllByText('WETH').first().click({ scrollBehavior: false })
```

Comment:
Why does this query need changing?

Reply:
`findByText` throws if more than 1 element is found, 'WETH' is both the token name and symbol which doesn't work with the previous implementation